boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -580,4 +580,24 @@ TEST_F(
         0.0f, App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface));
 }
 
+// DCM-20
+TEST_F(
+    DcmStateMachineTest,
+    transition_to_fault_state_from_drive_state_if_motor_shutdown_error_detected)
```

Comment:
This is good. Can we also include tests where the error is introduced into the CanRx table instead of the error table (using `CanRx_SetSignal`)? So one test for receiving the BMS motor shutdown error over CAN, one for receiving FSM motor shutdown error over CAN, etc.

Reply:
In the CAN RX task, `Io_SharedErrorTable_SetErrorsFromCanMsg()` is called below `Io_CanRx_UpdateRxTableWithMessage()` which updates the error table with the latest CAN RX message in the queue. Therefore, I don't think updating the CanRx table will update the error table. I can add more to the test to check if motor shutdown errors from other boards result in a state transition.